include/dali/core/mm/cu_vm.h

```diff
@@ -228,8 +248,12 @@ T *Map(CUdeviceptr virt_addr, CUMemAllocation mem) {
   return Map<T>(virt_addr, mem.handle(), mem.size());
 }
 
+inline void Unmap(CUdeviceptr ptr, size_t size) {
+  CUDA_DTOR_CALL(cuMemUnmap(ptr, size));
```

Comment:
Is this always meant to run in a dtor? In case of doing it anywhere else, are we OK with not catching errors?

Reply:
It catches all errors except CUDA being unloaded.